src/player.js

```diff
@@ -1205,6 +1205,27 @@ export default class Player extends FakeEventTarget {
 
   // </editor-fold>
 
+  // <editor-fold desc="VR API">
+
+  isVr(): boolean {
+    return !!(this._config.sources.vr);
+  }
+
+  // </editor-fold>
+
+  // <editor-fold desc="Plugins API">
+
+  /**
+   * get a plugin instance
+   * @param {string} name - the plugin name
+   * @returns {?BasePlugin} - the plugin instance
+   */
+  getPlugin(name: string): ?BasePlugin {
```

Comment:
If we are already adding this API we need to add also something like
```js
isPluginActive(name) : boolean
```

Reply:
I see a case of exposing the plugins outside, think of cases where we are also using a 3rd party API that might be needed for application integration(hls.js needed by a 3rd party we don't implement, IMA is needed for MOAT integration etc.)

As we are adding VR specific stuff I would like to keep it focused and not add unplanned APIs.
@yairans add the `toggleVrStereoMode` here and refer it to the plugin directly - via the plugin manager, and change the UI to call `player.toggleVrStereoMode()`.
You can also remove the kaltura player bindings then.
Once we create the controller for VR we will change this implementation to use it.

Reply:
also, maybe toggle is not the best option, as it forces you to know the state, so you will have to also expose `isStereoModeSet`

Reply:
done